canvas_modules/common-canvas/src/common-properties/controls/datepicker/datepicker.jsx

```diff
@@ -64,7 +64,6 @@ class DatepickerControl extends React.Component {
 	}
 
 	render() {
-		const helperText = this.props.controller.getResource(`${this.props.control.name}.helper`, null);
```

Comment:
This would be a breaking change.  We might need to keep this but instead of having `null` using the control.helperText.